xbmc/pictures/metadata/ImageMetadataParser.cpp

```diff
@@ -75,27 +83,28 @@ std::unique_ptr<ImageMetadata> CImageMetadataParser::ExtractMetadata(const std::
   CImageMetadataParser parser;
 
   // extract metadata
-  parser.ExtractCommonMetadata(image);
+  parser.ExtractCommonMetadata(*image);
 
   parser.ExtractExif(image->exifData());
   parser.ExtractIPTC(image->iptcData());
 
   return std::move(parser.m_imageMetadata);
 }
 
-void CImageMetadataParser::ExtractCommonMetadata(std::unique_ptr<Exiv2::Image>& image)
+void CImageMetadataParser::ExtractCommonMetadata(Exiv2::Image& image)
 {
   //! TODO: all these elements are generic should be moved out of the exif struct
-  m_imageMetadata->height = image->pixelHeight();
-  m_imageMetadata->width = image->pixelWidth();
-  m_imageMetadata->fileComment = image->comment();
-
-  if (image->imageType() == Exiv2::ImageType::jpeg)
+  m_imageMetadata->height = image.pixelHeight();
+  m_imageMetadata->width = image.pixelWidth();
+  m_imageMetadata->fileComment = image.comment();
+#if !defined(OLD_EXIV_API)
+  if (image.imageType() == Exiv2::ImageType::jpeg)
```

Comment:
This stuff here requires exactly 0.28.2 or above not just 0.28.0

Reply:
Done.